tests/test_formatting.py

```diff
@@ -24,8 +24,13 @@ def test_code_style(source_root):
     py_files_ert_gui = get_py_files(os.path.join(source_root, "ert_gui"))
     py_files_tests = get_py_files(os.path.join(source_root, "tests"))
     py_files_test_data = get_py_files(os.path.join(source_root, "test-data"))
+    py_files_ert3 = get_py_files(os.path.join(source_root, "ert3"))
```

Comment:
Hmm, I was actually under the impression that one had to explicitly opt out if one did not want to be style checked. But apparently I remember it wrong!